recipes/scip/all/conandata.yml

```diff
@@ -11,3 +14,24 @@ sources:
   "8.0.3":
     url: "https://github.com/scipopt/scip/archive/refs/tags/v803.tar.gz"
     sha256: "fe7636f8165a8c9298ff55ed3220d084d4ea31ba9b69d2733beec53e0e4335d6"
+patches:
+  "9.2.0":
+    - patch_file: "patches/0001-bliss-include-dir.patch"
+      patch_description: "Change hard-coded paths to conan includes"
+      patch_type: "conan"
+version_mappings:
+  "9.2.0":
+    soplex: "7.1.2"
```

Comment:
mmm this version does not seem to exist

```
  soplex
    soplex/6.0.3
    soplex/6.0.4
    soplex/7.0.0
    soplex/7.0.1
    soplex/7.1.0
    soplex/7.1.1
```

Reply:
This MR depends on the one adding scip/7.1.2

Reply:
Excellent - relaunching this so that it gets built now